tests/unit/test_kokkos_1d_commons.h

```diff
@@ -62,36 +66,43 @@ static void compare1d(ViewT const& k1, ViewT const& k2) {
   compareInner1d<ViewT,1>(k1,k2);
 }
 
+template <typename ViewT>
+static void compare1dDynamic(ViewT const& k1, ViewT const& k2) {
+  compareBasic(k1,k2);
+  std::cout << "compare1dDynamic: " << k1.extent(0) << "," << k2.extent(0) << "\n";
+  EXPECT_EQ(k1.extent(0), k2.extent(0));
+
+  for (typename ViewT::size_type i = 0; i < k1.extent(0); i++) {
+    EXPECT_EQ(k1.operator()(i), k2.operator()(i));
```

Comment:
Out of curiosity, why explicit `operator()` call instead of `EXPECT_EQ(k1(i), k2(i));` ?

Reply:
Monkey see, monkey do. I was copying the existing code from `compareInner`